Makefile

```diff
@@ -326,20 +335,28 @@ test-single-gtest-case: ## Run a single GoogleTest test case: `make test-single-
 		--gtest_filter=*${TEST_CASE_FILTER}* --gtest_repeat=${NUM_REPEATS__} $${break_on_failure_opt}";
 
 .PHONY: clean
-clean: ## Clean Concord-BFT build directory
+clean: create-ccache-folder ## Clean Concord-BFT build directory
 	docker run ${BASIC_RUN_PARAMS} \
 		${CONCORD_BFT_CONTAINER_SHELL} -c \
 		"rm -rf ${CONCORD_BFT_BUILD_DIR}"
 
 .PHONY: clean-all
-clean-all: ## Clean Concord-BFT build directory and any other untracked/ignored files.For a 'dry run' use DRY_RUN=TRUE, for an interactive run use INTER=TRUE.
+clean-all: ## Clean Concord-BFT build directory and any other untracked/ignored files. Does not include ccache cache folder.For a 'dry run' use DRY_RUN=TRUE, for an interactive run use INTER=TRUE.
```

Comment:
I believe that this should also remove the ccache (same effect as removing the build directory), otherwise the name is misleading.

Reply:
That's a bit of a problem. What if I just want to clean the whole local folder, build and any leftovers and not clean ccache? This is usually what I want. I agree name might be a bit misleading but the idea is to clean everything inside the current root project folder, and ccache cache is not under that folder, so it's a matter of interpertation - everything is cleaned under concord-bft, we don't delete /tmp/* as well, right?

ccache deletion should be explicit by running clean-ccache-cache or rm -rf $HOME/.ccache/*